Android/app/src/androidTest/java/io/github/project_travel_mate/ApplicationTest.java

```diff
@@ -1,13 +0,0 @@
-package io.github.project_travel_mate;
```

Comment:
why did you delete this?

Reply:
Because it was deprecated and gives me an error while building the project.

Reply:
There were no build errors before. there shouldnt be a need to remove this